.github/workflows/release-embedding-sdk.yml

```diff
@@ -0,0 +1,180 @@
+name: Release Metabase Embedding SDK for React
+
+on:
+  workflow_dispatch:
+    inputs:
+      sdk_version:
+        description: 'SDK version (e.g. 0.1.3)'
+        type: string
+        required: true
+      commit:
+        description: 'Optional full-length commit SHA-1 hash'
+  # Force trigger the workflow
+  pull_request:
+    branches:
+      - 42498-automate-sdk-release
+
+
+concurrency:
+  # We want to ensure only one job is running at a time because
+  # there could be a conflict when updating the readme file.
+  group: ${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  git-tag:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 20
+    steps:
+      - name: Check out the code using the provided commit, or HEAD
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.commit }}
+
+      - name: Setup git user
+        run: |
+          git config --global user.email "metabase-bot@metabase.com"
+          git config --global user.name "Metabase bot"
+
+      - name: Create a new git tag
+        run: |
+          git tag embedding-sdk-${{ inputs.sdk_version }}
+
+      - name: Push the new tag
+        run: |
+          git push origin embedding-sdk-${{ inputs.sdk_version }}
+      - if: failure()
+        run: echo "Make sure the tag 'embedding-sdk-${{ inputs.sdk_version }}' doesn't exist."
+
+  test:
+    needs: git-tag
+    runs-on: ubuntu-22.04
+    timeout-minutes: 20
+    steps:
+      - name: Check out the code using the provided commit, or HEAD
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.commit }}
+
+      - name: Prepare front-end environment
+        uses: ./.github/actions/prepare-frontend
+
+      - name: Prepare back-end environment
+        uses: ./.github/actions/prepare-backend
+        with:
+          m2-cache-key: "cljs"
+
+      - name: Run unit tests
+        run: yarn embedding-sdk:test-unit
+
+  build-sdk:
```

Comment:
I forgot to add the step to bump the SDK template version. This will cause the built SDK to still have the old published version. I'll work on this tomorrow.